Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/ExposureKeyDomain.kt

```diff
@@ -0,0 +1,44 @@
+package de.rki.coronawarnapp.submission
+
+import KeyExportFormat
+import com.google.android.gms.nearby.exposurenotification.TemporaryExposureKey
+import com.google.protobuf.ByteString
+
+class ExposureKeyDomain {
```

Comment:
Does `Domain` fit here? Isn't this used like a `ExposureKeyConverter`?

Reply:
no, this does class does not do converting

Reply:
it contains logic for exposure key history

Reply:
A Domain does not contain logic imo. I agree though that this is not pure conversion. Logic is used for interpreting keys that come out of the API and correctly assign Risk Levels as well, so maybe we should decouple and/or rename accordingly to purpose